openedx/features/caliper_tracking/caliper_config.py

```diff
@@ -8,10 +8,16 @@
     'edx.bookmark.added': ctf.edx_bookmark_added,
     'edx.bookmark.listed': ctf.edx_bookmark_listed,
     'edx.ui.lms.link_clicked': ctf.edx_ui_lms_link_clicked,
-    'edx.course.enrollment.activated': ctf.edx_course_enrollment_activated,
-    'edx.course.enrollment.deactivated': ctf.edx_course_enrollment_deactivated,
-    'edx.course.enrollment.mode_changed': ctf.edx_course_enrollment_mode_changed,
-    'edx.course.enrollment.upgrade.clicked': ctf.edx_course_enrollment_upgrade_clicked,
+    'edx.course.enrollment.activated':
+        ctf.edx_course_enrollment_activated,
+    'edx.course.enrollment.deactivated':
+        ctf.edx_course_enrollment_deactivated,
+    'edx.course.enrollment.mode_changed':
+        ctf.edx_course_enrollment_mode_changed,
+    'edx.course.enrollment.upgrade.clicked':
+        ctf.edx_course_enrollment_upgrade_clicked,
     'edx.course.tool.accessed': ctf.edx_course_tool_accessed,
     'problem_show': ctf.problem_show,
+    'edx.ui.lms.sequence.next_selected':
+        ctf.edx_ui_lms_sequence_next_selected,
 }
```

Comment:
Please align them in one line there is no need to move them in next line **edX** line limit is 120 ignore "pep8speasks" line length error for now.